core/src/test/java/edu/jhuapl/dorset/http/HttpClientTest.java

```diff
@@ -0,0 +1,75 @@
+package edu.jhuapl.dorset.http;
```

Comment:
Insert "Copyright 2016 The Johns Hopkins University Applied Physics Laboratory LLC All rights reserved..." comment here.